tests/Feature/BookTest.php

```diff
@@ -249,4 +249,44 @@ public function test_getBookGenre_success(): void
                     });
             });
     }
+
+    public function test_addBook_success(): void
+    {
+        Genre::factory()->create();
+
+        $testData = [
+            'title' => 'A boooooook',
+            'author' => 'An author',
+            'genre_id' => 1
+        ];
+
+        $response = $this->postJson('/api/books', $testData);
+
+        $response->assertStatus(201)
+        ->assertJson(function (AssertableJson $json) {
+            $json->hasAll(['message', 'success']);
+        });
+
+        $this->assertDatabaseHas('books', $testData );
+    }
+
+    public function test_addBook_failure(): void
```

Comment:
Can you add an extra test to test the validation a bit more - by providing invalid data for all fields